Rubberduck.Refactorings/Common/ParseTreeValue/ComparableDateValue.cs

```diff
@@ -97,7 +97,7 @@ public static bool TryParse(string valueText, out ComparableDateValue value)
             {
                 return false;
             }
-            catch (Exception e)
+            catch (Exception)
```

Comment:
While this `catch` block basically swallows any exceptions, I do wonder if we need to at least trace thrown exceptions? The catch-all block bothers me because they can help hide exceptions that should not be swallowed or otherwise hide bugs/flaws in the code.